### PR TITLE
Ajout de bijoux

### PR DESCRIPTION
**Pull request by vinyll**
- Refactoring du système de calculs
- formatage des résultats avec arrondi > 50
- refactoring de la fonction de templating pour accepter des fonctions()

### CODE REVIEW

File: bijoux/index.html [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+</header>
+
+<main id=container></main>
+<footer></footer>
+
+<script>
+  window.conf = {
+      product: 'bijoux'
+  }
+  function redirectTo(target) {
```
Comment (davidbgk) [inline bijoux/index.html]: What about moving it to `main.js`?

File: js/main.js [hunk 2]
```diff
@@ -38,35 +38,40 @@ var round = 0
...
-  get amount () {
-    return this._amount
+  amountWithTva: function () {
+    return this.addPercentage(this.amount, 20)
   },
 
-  set amount (value) {
-    this._amount = value
-    this.addTva(20)
+  amountWithFees: function () {
```
Comment (davidbgk) [inline js/main.js:59]: Maybe add a comment to explain that we have to apply VAT to fees?

File: js/main.js [hunk 3]
```diff
@@ -38,35 +38,40 @@ var round = 0
...
 function templatize (str, context) {
-  return str.replace(/{[\w.]*}/g, function (match) {
+  return str.replace(/{[\w.\(\)]*}/g, function (match) {
     var varName = match.substr(1, match.length - 2)
     return context ? context[varName] : eval(varName)
   })
 }
 
+function formatNum(value) {
+  if(Number(value) > 50) {
```
Comment (davidbgk) [inline js/main.js]: Is that required to filter only values > 50?

File: parfums/index.html [hunk 4]
```diff
@@ -52,10 +52,10 @@
 
 <template id=tva-zero data-last>
   <h1>
-    Votre achat vous coûtera {calculator.amount} Euros au total (<abbr title="Toutes Taxes Comprises">TTC</abbr>).
+    Votre achat vous coûtera {formatNum(calculator.amount)} Euros au total (<abbr title="Toutes Taxes Comprises">TTC</abbr>).
```
Comment (davidbgk) [inline parfums/index.html]: Consistency with `calculator.amount()` would be great, don't you think?

**Review by davidbgk — COMMENTED** [review]
Thanks for finishing it 👍

File: js/main.js (hunk 3 above)
Comment (vinyll) [reply js/main.js]: We talked about that and it was suggested to round when it's over 50. Not sure how this discussions ended. Let me ask.

**Comment by vinyll** [comment]
The `formatNum()` now converts _123.39_ into _123,40_ (round, double decimal, comma separated).
`calculator.amount` is now `calculator.amount()`

**Review by davidbgk — APPROVED** [review]

**Pull request merged**